unity-environment/Assets/ML-Agents/Scripts/Communicator/EnvironmentParameters.cs

```diff
@@ -0,0 +1,168 @@
+// <auto-generated>
```

Comment:
Just a thought, is there a way not to distribute auto generated files with the repo, but instead have them automatically generated when needed?

Reply:
This will require the end user to install protoc if we do not provide these files. It might introduce some new friction, but there might be a way to compress and decompress them. We can even think of putting them into a dll.

Reply:
Fair point. Are all the generated files in the Communicator subdirectory, and only generated files reside there?

Reply:
Right, the files generated by protobuf are in `unity-environment/Assets/ML-Agents/Scripts/Communicator` and in `python/communicator`